Orange/widgets/regression/owsgdregression.py

```diff
@@ -139,7 +144,8 @@ def set_data(self, data):
             self.apply()
 
     def apply(self):
-        loss = ["squared_loss", "huber", "epsilon_insensitive", "squared_epsilon_insensitive"][self.loss_function]
+        loss = ["squared_loss", "huber", "epsilon_insensitive",
+                "squared_epsilon_insensitive"][self.loss_function]
         penalty = ["l1", "l2", "elasticnet"][self.penalty_type]
         learning_rate = ["invscaling", "constant"][self.learning_rate]
```

Comment:
I meant something like:
```py
PENALTIES = (
    ('Absolute norm (L1)',      'l1'),
    ('Euclidean norm (L2)',     'l2'),
    ('Elastic net (L1 and L2)', 'elasticnet'),
)
```
Now, there's the presentational form in the first column, and the library form in the second. But they are all defined together and in one single place.

Likewise for losses and learning rates.

This is, technically, out of scope of this PR, but the refactoring would still be welcome if in its own commit.